pkg/cmd/roachtest/cdc.go

```diff
@@ -157,6 +159,156 @@ func cdcBasicTest(ctx context.Context, t *test, c *cluster, args cdcTestArgs) {
 	verifier.assertValid(t)
 }
 
+func runCDCBank(ctx context.Context, t *test, c *cluster) {
+	// Make the logs dir on every node to work around the `roachprod get logs`
+	// spam.
+	c.Run(ctx, c.All(), `mkdir logs`)
+
+	crdbNodes, workloadNode, kafkaNode := c.Range(1, c.nodes-1), c.Node(c.nodes), c.Node(c.nodes)
+	c.Put(ctx, cockroach, "./cockroach", crdbNodes)
+	c.Put(ctx, workload, "./workload", workloadNode)
+	c.Start(ctx, t, crdbNodes)
+	kafka := kafkaManager{
+		c:     c,
+		nodes: kafkaNode,
+	}
+	kafka.install(ctx)
+	kafka.start(ctx)
+	defer kafka.stop(ctx)
+
+	c.Run(ctx, workloadNode, `./workload init bank {pgurl:0}`)
+	db := c.Conn(ctx, 1)
+	defer stopFeeds(db)
+
+	if _, err := db.Exec(
+		`SET CLUSTER SETTING changefeed.experimental_poll_interval = '0ns'`,
+	); err != nil {
+		t.Fatal(err)
+	}
+	var jobID string
+	if err := db.QueryRow(
+		`CREATE CHANGEFEED FOR bank.bank INTO $1 WITH updated, resolved`, kafka.sinkURL(ctx),
+	).Scan(&jobID); err != nil {
+		t.Fatal(err)
+	}
+
+	t.Status("running workload")
+	workloadCtx, workloadCancel := context.WithCancel(ctx)
+	m := newMonitor(workloadCtx, c, crdbNodes)
+	var doneAtomic int64
+	m.Go(func(ctx context.Context) error {
+		err := c.RunE(ctx, workloadNode, `./workload run bank {pgurl:0}`)
+		if atomic.LoadInt64(&doneAtomic) > 0 {
+			return nil
+		}
+		return err
+	})
+	m.Go(func(ctx context.Context) error {
+		defer workloadCancel()
+		l, err := c.l.ChildLogger(`changefeed`)
```

Comment:
Apparently we should use `t.l.ChildLogger`, instead of `c.`.  Don't know why, was just informed of that in my last PR though.